SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1793,6 +1802,11 @@ void USpatialNetDriver::TickDispatch(float DeltaTime)
 			Connection->Flush();
 		}
 
+		if (RPCService.IsValid())
```

Comment:
For now I think we should be doing this after the actor updates have been propagated. So after the dispatcher processing.

Reply:
sure, done